libs/vkd3d/state.c

```diff
@@ -1665,6 +1660,15 @@ static void d3d12_pipeline_state_destroy_graphics(struct d3d12_pipeline_state *s
     VK_CALL(vkDestroyPipeline(device->vk_device, graphics->pipeline, NULL));
 }
 
+static void d3d12_pipeline_state_set_name(struct d3d12_pipeline_state *state, const char *name)
+{
+    if (d3d12_pipeline_state_is_compute(state))
```

Comment:
We should have a graphics pipeline as well, could use the "normal" pipeline. Fallback pipelines will be tricky though, can just ignore those.

Reply:
This isn't new in this commit, I think we should fix this separately.

Reply:
Ok.